openbas-api/src/main/java/io/openbas/utils/InjectUtils.java

```diff
@@ -46,11 +48,15 @@ public InjectStatusCommandLine getCommandsLinesFromInject(final Inject inject) {
                 : null,
             payload.getExternalId());
       } else {
-        // Inject comes from Caldera ability and tomorrow from other(s) Executor(s)
-        io.openbas.execution.Injector executor =
-            context.getBean(
-                injectorContract.getInjector().getType(), io.openbas.execution.Injector.class);
-        return executor.getCommandsLines(injectorContract.getId());
+        try {
+          // Inject comes from Caldera ability and tomorrow from other(s) Executor(s)
+          io.openbas.execution.Injector executor =
+              context.getBean(
+                  injectorContract.getInjector().getType(), io.openbas.execution.Injector.class);
+          return executor.getCommandsLines(injectorContract.getId());
+        } catch (NoSuchBeanDefinitionException e) {
```

Comment:
issue: You should log here that we are swallowing the exception

Reply:
True, I add some details.
Is this consistent with the idea you had?

Reply:
I'd probably escalate to Warning I suppose, it looks like a misconfiguration if the requested class is not available for loading.
Otherwise yes it's what I am referring to: no silently dropping an exception, so that we at least know from the logs that something is happening